cmd/kubeadm/app/constants/constants.go

```diff
@@ -481,9 +481,10 @@ var (
 
 	// SupportedEtcdVersion lists officially supported etcd versions with corresponding Kubernetes releases
 	SupportedEtcdVersion = map[uint8]string{
-		29: "3.5.15-0",
-		30: "3.5.15-0",
-		31: "3.5.15-0",
+		29: "3.5.16-0",
+		30: "3.5.16-0",
+		31: "3.5.16-0",
+		32: "3.5.16-0",
```

Comment:
Adding 32 makes the UT failing.
 https://github.com/kubernetes/kubernetes/blob/988235aa7fde259d6204cc2a1fb15a896468cf79/cmd/kubeadm/app/constants/constants_test.go#L101-L106